.github/workflows/test.yml

```
@@ -0,0 +1,53 @@
+name: test CI
+
+on: [pull_request]
+
+jobs:
+  pytest_with_coverage:
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    strategy:
+      matrix:
+        python-version: [3.7]
```

we're using python3.8
```suggestion
        python-version: [3.8]
```